test/compatibility_lifecycle/reference/versioned_feature_list.yaml

```diff
@@ -1739,6 +1739,10 @@
     lockToDefault: false
     preRelease: Beta
     version: "1.33"
+  - default: true
+    lockToDefault: false
```

Comment:
This doesn't match so you'll probably get a CI failure eventually. I think you need to generate this.